cogito/put_test.go

```diff
@@ -309,7 +309,7 @@ func TestPutterProcessInputDirNonExisting(t *testing.T) {
 func TestPutterSinks(t *testing.T) {
 	putter := cogito.NewPutter("dummy-API", hclog.NewNullLogger())
 
-	sinks := putter.Sinks()
+	sinks, _ := putter.Sinks()
```

Comment:
we should check that error is nil first.

Reply:
thanks, I addressed your point.